big_tests/tests/metrics_api_SUITE.erl

```diff
@@ -74,7 +74,7 @@ init_per_group(GroupName, Config) ->
     metrics_helper:prepare_by_all_metrics_are_global(Config, GroupName =:= all_metrics_are_global).
 
 end_per_group(GroupName, Config) ->
-    metrics_helper:finalise_by_all_metrics_are_global(Config, GroupName =:= all_metrics_are_global).
+    metrics_helper:finalize_by_all_metrics_are_global(Config, GroupName =:= all_metrics_are_global).
```

Comment:
I just feel like 'finalize' is much more commonly used in programming environments and libraries, so I thought it would improve the readability of the code a bit. Also, if I remember correctly, the Erlang standard library uses the atom `finalize` as an argument in some of its functions.

I agree, the example you found is indeed surprising.

Reply:
OK, this isn't a big issue for me. Although my first instinct would probably be to change American spelling back to British if I touched these lines :D